Assets/Game/Addons/ModSupport/ModManager.cs

```diff
@@ -158,6 +158,12 @@ void Start()
             }
         }
 
+        void OnDestroy()
+        {
+            foreach (Mod mod in mods)
+                mod.UnloadAssetBundle(unloadAllObjects: true);
+        }
```

Comment:
This line needs to be added here in `OnDestroy()`. It makes sense to add this line from #2447 but that could cause a merge conflict between these 2 PR
```
void OnDestroy()
{
    StateManager.OnStateChange -= StateManager_OnStateChange;
    /* code this PR adds goes here */
}
```
I am writing this down here to not forget about it